packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { which } from 'shelljs';
```

Comment:
Using `shelljs` allows us to have a cross platform way of figuring out if the cli is available in the system.